website/www/site/i18n/home/en.yaml

```diff
@@ -35,15 +35,15 @@
 - id: home-ctas-title
   translation: "Get started"
 - id: home-model-title1
-  translation: "How Does it work?"
+  translation: "How Does It Work?"
 - id: home-model-title2
-  translation: "Write it in your language of choice - run it anywhere"
+  translation: "You Choose Your Favorite Environment and Programming Language!"
```

Comment:
I'd recommend sentence case instead of title case here.

Reply:
ditto